server/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Moving from gradle to maven should be done in a separate branch imo

Reply:
That's true, however since this was the first branch that we operated on, I thought that moving to maven which enabled convenient testing wouldn't cause a lot of mess. It was quite necessary in order to complete the core part of our model.